src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.Swagger/Internal/MessageDescriptorHelpers.cs

```diff
@@ -8,6 +8,24 @@ namespace Microsoft.AspNetCore.Grpc.Swagger.Internal;
 internal static class MessageDescriptorHelpers
 {
     public static Type ResolveFieldType(FieldDescriptor field)
+    {
+        if (field.IsMap)
+        {
+            var mapFields = field.MessageType.Fields.InFieldNumberOrder();
+            var valueType = GetTypeFromFieldType(mapFields[1]);
+            return typeof(IDictionary<,>).MakeGenericType(typeof(string), valueType);
+        }
+        else if (field.IsRepeated)
+        {
+            return typeof(IList<>).MakeGenericType(GetTypeFromFieldType(field));
```

Comment:
I see this changed from `List` to `List`. Is that intentional?

Reply:
Not intentional, but it doesn't matter. Swashbuckle generates the same output for IList and List.